### ISSUE TITLE
JSON structure from Front end => BackEnd for CRUD

### ISSUE THREAD

**Issue by suBOTai3** [koorzenb/koorzenb-TimeLogger#2]
```Javascript 
{
  "description" : "time that was logged",
  "startDate" : "2022-04-04T134500.000"
  "endDate" : "2022-04-04T143000.000"
}```

**Comment by koorzenb** [comment]
U guess 'id' is implied?

**Comment by suBOTai3** [comment]
So another thing...

For **/GET** requests you dont send a body. The resource is included request url eg

**/GET** /api/tasks/{taskId}

**Comment by suBOTai3** [comment]
I'll mockup a sample set of requests on Postman and share the collection... We can then iterate on the requests and see what works for you

**Comment by koorzenb** [comment]
Sure.

I guess then the id is generated by a backend process?

**Comment by suBOTai3** [comment]
Yes so initially when a resource is created (POST), the id is generated in the backend and returned as part of the 201 Created result (above)

Check out this article

https://www.w3schools.com/tags/ref_httpmethods.asp#:~:text=The%20difference%20between%20POST%20and,the%20same%20resource%20multiple%20times.
